### ISSUE TITLE
about training datas prepared

### ISSUE THREAD

**Issue by dreambander** [eragonruan/text-detection-ctpn#188]
hello , i have some driving license pictures, and i want use them to train new model . but i don't know how to deal my picture to send to model, whether directory read images path to split_label.py and ToVoc.py ,thanks.

**Comment by eragonruan** [comment]
@dreambander hi,first prepare you have to label your dataset, the ground truth should be like 
x1,y1,x2,y2,x3,y3,x4,y4. And then, follow readme to convert the training data and train the model.

**Comment by kellenf** [comment]
@dreambander [This](https://kellenf.github.io/#/user/blog/details/285821184eb3766c7d2e0b869e237be4) can help you

**Comment by dreambander** [comment]
thank you so much, I understand.

**Comment by forever1love0612** [comment]
May you give the format of text files in gt_path in detail? I am in an urgent need of it.

**Comment by forever1love0612** [comment]
May you give the format of text files in gt_path in detail? I am in an urgent need of it. @eragonruan
